AdyenUI/AdyenElementStyles.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+public struct AdyenElementStyles {
+    
+    public var button: AdyenButtonTypes
+    public var label: LabelStyle
```

Comment:
Idea: Do you want to maybe have multiple `LabelTypes` the same way you have multiple `ButtonTypes`?
e.g. `TitleStyle`, `CaptionStyle`, ...